erpnext/quality_management/doctype/customer_feedback/test_customer_feedback.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe and Contributors
+# See license.txt
+from __future__ import unicode_literals
+
+import frappe
+import unittest
+
+class TestCustomerFeedback(unittest.TestCase):
+	def test_customer_feedback(self):
+		create_feedback_template()
+		test_create_feedback = create_feedback()
+		test_get_feedback = get_feedback()
+		self.assertEquals(test_create_feedback.name, test_get_feedback.name)
+
+def create_feedback():
+	feedback = frappe.get_doc({
+		"doctype": "Customer Feedback",
+		"template": "FDBK-TMPL-_Test Customer Feedback Template",
+		"date": ""+ frappe.utils.nowdate() +""
+	})
+	feedback_exist = frappe.get_list("Customer Feedback", filters={"date": ""+ feedback.date +""})
```

Comment:
You can use `frappe.db.exists()` to do these checks instead, and save a little on testing time.